README.md

```diff
@@ -86,6 +86,27 @@ If you have any questions about Zodiac, join the [Gnosis Guild Discord](https://
 
 The [Zodiac documentation](https://gnosis.github.io/zodiac/) offers tutorials on how to use the Zodiac App, and detailed developer resources on how to build your own Zodiac module, modifier, or guard will be available soon.
 
+### Zodiac Banners and Badges
+Show your support for Zodiac by including our "Built By Zodiac" badges or banners on your site or in your repository. Check out the previews below and download the ones that you like best. Please wrap the image with a link to [Zodiac](https://gnosis.github.io/zodiac/).
+
```

Comment:
would be really nice to have a markdown ready to copy and paste version for a couple of the badges, maybe just the light and dark monochromatic ones. something like

`[![White on black banner](https://raw.githubusercontent.com/gnosis/zodiac/master/branding/zodiac-badge-black-white.svg)](https://gnosis.github.io/zodiac/)